### ISSUE TITLE
Example shows error message on import

### ISSUE THREAD

**Issue by SaschaHeyer** [kubeflow/fairing#304]
/kind bug

**What steps did you take and what happened:**
1. Cloned the GitHub examples into a Kubeflow Notebook
1. Opened the following example /notebook/kubeflow/pipelines/notebooks/fairing/examples/train_job_api/main.ipynb
1. Run the example

I get the following error message:
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-4-99bd2575c1ac> in <module>
      2 from fairing import TrainJob
      3 from fairing.backends import KubeflowGKEBackend
----> 4 from fairing.kubernetes.utils import get_resource_mutator

ModuleNotFoundError: No module named 'fairing.kubernetes.utils'
```

**What did you expect to happen:**
No error message

**Environment:**

- Fairing version: ```module 'fairing' has no attribute '__version__'```
- Kubeflow version: build version v0.5.0

**Comment by jinchihe** [comment]
@SaschaHeyer I think the `fairing/kubernetes/utils.py` is added recently by #262 , not included in the last official released 0.5.3, but that will be included next release. You can install from github or copy the file from github and then try again.

**Comment by SaschaHeyer** [comment]
@jinchihe great thank you
